module_25_Python/3_number_of_lines.py

```diff
@@ -0,0 +1,27 @@
+"""
+Задача 3. Количество строк
+Реализуйте функцию-генератор, которая берёт все питоновские файлы
+в директории и вычисляет количество строк в каждом файле,
+игнорируя пустые строки и строки комментариев.
+По итогу функция-генератор должна с помощью yield каждый раз возвращать количество строк
+в очередном файле.
+"""
+
+import os
+
+
+def gen_number_of_file_lines(directory: str):
+    for file in os.listdir(directory):
+        lines_amount = 0
+        if file.endswith('.py'):
+            with open(file, 'r') as python_file:
+                for line in python_file:
+                    if not line == '\n' and not line.strip().startswith(('#', '"')):
```

Comment:
What about lines with just whitespaces, i.e.
```py
_____\t\n
```
(5 spaces, one tab character and a newline)

Reply:
done